packages/server/src/utility/algoliaIndex.ts

```diff
@@ -0,0 +1,22 @@
+import * as dotenv from "dotenv";
+
+import algoliaSearch from "algoliasearch";
+
+dotenv.config();
+
+const {
+  ALGOLIA_APP_ID,
+  ALGOLIA_WRITE_API_KEY,
```

Comment:
I created a new key for this and added to Heroku. We can confirm that key works after this is deployed.